### ISSUE TITLE
Add option to create with query parameters

### ISSUE THREAD

**Issue by patcon** [steveklabnik/issue2pr#5]
Thinking that if an issue could be converted by hitting a url with query params, then someone could pretty easily write a [userscript](http://userscripts.org/) to add a button onto the repo's page on github. (This would only work seamlessly after connecting your github account, of course.)

Idea stemmed from isaacs/github#93

**Comment by steveklabnik** [comment]
I like this idea, bur GET shouldnt be used to modify server state...

**Comment by patcon** [comment]
You're absolutely right. It felt wrong, so I should have thought about it more. What about just autocompleting the fields? That feels appropriately conservative :)

EDIT: So userscript flow could be:
1) detects that we are on an issue page
2) adds nicely styled button for "convert to PR"
3) On click, polls github API to find candidate branches (in this repo and your forks?)
4) When user selects one, it redirects them to the appropriate issue2pr url with query params filling in the fields correctly
5) User clicks button to pull requesterize the issue (and is redirected back to original github page?)

Note, that I haven't done much work with script monkey stuff, so perhaps it's not so simple. The last point would also require a query param for the userscript to store the referring page on github.

**Comment by steveklabnik** [comment]
Yes! Thatd be cool.

**Comment by patcon** [comment]
Oops. Updated my comment above -- not sure if it notifies you!
